sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/FieldBuilder.java

```diff
@@ -38,28 +43,26 @@
         SUPPORTED_NONE_PARAMETERIZED_TYPE.put(String.class, SearchFieldDataType.STRING);
         SUPPORTED_NONE_PARAMETERIZED_TYPE.put(Date.class, SearchFieldDataType.DATE_TIME_OFFSET);
         SUPPORTED_NONE_PARAMETERIZED_TYPE.put(OffsetDateTime.class, SearchFieldDataType.DATE_TIME_OFFSET);
-//        SUPPORTED_NONE_PARAMETERIZED_TYPE.put(PointGeometry.class, SearchFieldDataType.GEOGRAPHY_POINT);
+        SUPPORTED_NONE_PARAMETERIZED_TYPE.put(PointGeometry.class, SearchFieldDataType.GEOGRAPHY_POINT);
     }
 
-    private static final List<Class<?>> UNSUPPORTED_TYPES = Arrays.asList(Byte.class,
-        CharSequence.class,
-        Character.class,
-        char.class,
-        Float.class,
-        float.class,
-        Short.class,
-        short.class);
+    private static final List<Class<?>> UNSUPPORTED_TYPES = Arrays.asList(byte.class, Byte.class,
+        CharSequence.class, Character.class, char.class,
```

Comment:
Any reason we can't support `CharSequence` by using its `toString` method which returns its value as a `String`? I believe Jackson and GSON will handle `CharSequence` similar to a `String`.

Reply:
Yes, I think this one can move to the supported type mapping if we all agree to it.

Reply:
Moved to Supported search date type mapper